src/SharpBrick.PoweredUp/Hubs/TwoPortHandset.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.Logging;
+
+using SharpBrick.PoweredUp.Devices;
+using SharpBrick.PoweredUp.Protocol;
+
+using System;
+
+namespace SharpBrick.PoweredUp
+{
+    public class TwoPortHandset : Hub
```

Comment:
Is TwoPortHandset the canoncial name used in other implementations?

Reply:
I simply used the name that was already available in SystemType 😀

The device itself announces itself simply as 'Handset', so I guess TwoPortHandset makes sense?

Reply:
I checked other libraries, and most refer to the remote and the train hub as `Powered Up Remote` or `Remote` and `Powered Up Hub` or simply `Hub` respectively.

Since the train hub is called the TwoPortHub and it's bluetooth name is `HUB NO.4`, the remote bluetooth name is `Handset` so I think TwoPortHandset is in order. I'll mark this as resolved for now. If you think otherwise, let me know and I can change the name.